### DISCUSSION TITLE
现在引入designable至生产环境合适吗

### DISCUSSION THREAD

**Discussion by aidening** [alibaba/designable#92] [General]
我们基于 formily 1.0 做了一版 表单设计器，现在想升级 formily 2.0，想直接引入 designable，并基于 designable 做一些扩展

需要扩展的点会涉及到 最右侧的配置面板，以及响应数据等等

请问进行这一系列的操作难度大吗？

进行扩展时，有足够的口子进行操作吗？

请问有没有 designable 相关文档呢？

**Comment by janryWang** [comment]
没有文档，具体怎么用看源码。扩展性没问题的

**Reply by aidening** [reply]
好的，感谢
